testing/test_server.py

```diff
@@ -20,8 +21,13 @@
 Send a POST request:
     curl -d "foo=bar&bin=baz" http://localhost
 """
+from SocketServer import ThreadingMixIn
```

Comment:
Do we need a way for MLDB to run something (eg a function) in multiple threads?  It would be nice to avoid having to jump outside of the normal MLDB functionality for this.